README.md

```diff
@@ -27,7 +27,10 @@ About this section. Optional. Keep this short and focus on the list.
 
 ## Other
 
+- [`DisFin` Discord Rich Presence for Jellyfin Activity](https://github.com/IrishBumfuzzle/DisFin)
```

Comment:
```suggestion
```

Reply:
[DisFin](https://github.com/IrishBumfuzzle/DisFin) could also be replaced with [MBCord](https://github.com/oonqt/MBCord) as an alternative. Does essentially the same thing. Let me know what you think otherwise we can scrap that line entirely.

Reply:
I'm not a big fan of MBCord either. Issues seem to be closed without being fixed, and the maintainer doesn't really take the users seriously either, as you can see from a couple of GitHub-issues 🫤.

Reply:
Fair enough, I've removed the line.